content/source/docs/cloud/run/cli.html.md

```diff
@@ -41,63 +41,50 @@ To supplement these remote operations, you can also use the optional [Terraform
 
 ~> **Note:** The [Structured Run Output](../workspaces/settings.html#user-interface) user interface will not apply to runs executed using the CLI-driven workflow, regardless of the setting in the Terraform Cloud workspace.
 
-## Remote Backend Configuration
+## Configuration
 
-To configure the remote backend, a stanza needs to be added to the Terraform configuration. It must specify the `remote` backend, the name of a Terraform Cloud organization, and the workspace(s) to use. The example below uses one workspace; see [the remote backend documentation][remote] for more details.
+To enable the CLI-driven workflow, you must:
 
-```hcl
-terraform {
-  backend "remote" {
-    organization = "my-org"
+1. Add the `cloud` block to your Terraform configuration. For example:
 
-    workspaces {
-      name = "my-app-dev"
+    ```
+    terraform {
+      cloud {
+        organization = "my-org"
+        workspaces {
+          tags = ["networking"]
+        }
+      }
     }
-  }
-}
-```
-
-Next, run `terraform login` to authenticate with Terraform Cloud. Alternatively, you can [manually configure credentials in the CLI config file](/docs/cli/config/config-file.html#credentials).
-
-The backend can be initialized with `terraform init`.
-
-```
-$ terraform init
-
-Initializing the backend...
+    ```
 
-Initializing provider plugins...
+1. Run `terraform init`.
 
-Terraform has been successfully initialized!
+Refer to [Using Terraform Cloud](cloud) for more details about how to initialize and configure the integration.
 
-You may now begin working with Terraform. Try running "terraform plan" to see
-any changes that are required for your infrastructure. All Terraform commands
-should now work.
-
-If you ever set or change modules or backend configuration for Terraform,
-rerun this command to reinitialize your working directory. If you forget, other
-commands will detect it and remind you to do so if necessary.
-```
+~> **Note**: The `cloud` block is available in Terraform v1.1 and later and Terraform Enterprise v202201 and later. Previous versions can use the [`remote` backend](/docs/language/settings/backends/remote.html) to configure the CLI workflow and migrate state.
```

Comment:
@im2nguyen This is something you've called out in my PRs before (the opposite - to change -> to ~> because I'm assuming that's what we do on learn). What are your thoughts here? I agree with Judith that the docs do have a convention of doing notes like -> BUT if we're moving toward a unified experience, maybe we should begin mirroring what we do on Learn?   Casy-by-case maybe isn't the right way to do it...but then again we should start somewhere? Anyway, lmk what you think and once we have consensus, I'll update accordingly!

Reply:
I think for now, let's go with Judith's suggestion to make everything consistent (in docs, notes are `->`)

when we have bandwidth, we can create a new PR that replaces all `-> **Note:**` to `~> **Note:**

remember to also make the colon bold!

Reply:
I think this note works better right after the first bullet since note is specifically targeted at the `cloud` block

![image](https://user-images.githubusercontent.com/7556325/144982103-072c031a-6581-42e7-a33e-e0fde7617891.png)

Reply:
this is an excellent suggestion - i'm going to move it up - thank you!!!

Reply:
FYI y'all - I fixed all the notes I could see :-) Thank you for the guidance and for calling that out!